Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs

```diff
@@ -529,25 +529,23 @@ private UserAc GetUser(ApplicationUser user)
         {
             if (user != null)
             {
-                var Roles = _userManager.GetRolesAsync(user).Result.First();
-                if (Roles.Equals(_stringConstant.Admin))
+                string roles = _userManager.GetRolesAsync(user).Result.First();
+                UserAc newUser = _mapperContext.Map<ApplicationUser, UserAc>(user);
+                if (String.Compare(roles, _stringConstant.Admin, true).Equals(0))
                 {
-                    var newUser = _mapperContext.Map<ApplicationUser, UserAc>(user);
-                    newUser.Role = Roles;
+                    newUser.Role = roles;
                     return newUser;
                 }
-                if (Roles.Equals(_stringConstant.Employee))
+                if (String.Compare(roles, _stringConstant.Employee, true).Equals(0))
```

Comment:
I thought that Equals is better and I had followed the same method in Bot Implementation too.